packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1373,18 +1374,18 @@ export async function saveFile(doc: vscode.TextDocument, datasetProvider: api.IZ
     );
     label = label.toUpperCase().trim();
     globals.LOG.debug(localize("saveFile.log.debug.saving", "Saving file ") + label);
-    if (!label.includes("(")) {
-        try {
-            // Checks if file still exists on server
-            const response = await ZoweExplorerApiRegister.getMvsApi(profile).dataSet(label, { responseTimeout: profile.profile?.responseTimeout });
-            if (!response.apiResponse.items.length) {
-                return api.Gui.errorMessage(
-                    localize("saveFile.error.saveFailed", "Data set failed to save. Data set may have been deleted on mainframe.")
-                );
-            }
-        } catch (err) {
-            await errorHandling(err, sesName, err.message);
+    const dsname = label.includes("(") ? label.slice(0, label.indexOf("(")) : label;
```

Comment:
Smart way to get the DS name.
Curious if this should be made a common utility (either in the CLI or the Zowe Explorer API)?
I know I had to create a similar one for the FTP plugin not long ago 😋 

---

Unrelated question, _what if_ we decide to change the label in order to add more complex descriptions (instead of using the `description` field)… will this be trimming undesired portions of the label?